src/betterbib/cli/doi2bibtex.py

```diff
@@ -18,20 +21,54 @@ def _get_version_text():
     )
 
 
+def _create_key_for_entry(entry):
```

Comment:
How about adding a test for this? I want to see what this looks like for a bunch of bibtex entries.